FunctionSignatureMap.php

```diff
@@ -10276,28 +10281,30 @@
 'Redis::rpoplpush' => ['string', 'srcKey'=>'string', 'dstKey'=>'string'],
 'Redis::rPush' => ['bool|int', 'key'=>'string', 'value1'=>'string', 'value2='=>'string', 'valueN='=>'string'],
 'Redis::rPushx' => ['int', 'key'=>'string', 'value'=>'string'],
-'Redis::sAdd' => ['int', 'key'=>'string', 'value1'=>'string', 'value2='=>'string', 'valueN='=>'string'],
+'Redis::sAdd' => ['int|false', 'key'=>'string', 'value1'=>'string', 'value2='=>'string', 'valueN='=>'string'],
 'Redis::sAddArray' => ['bool', 'key'=>'string', 'values'=>'array'],
 'Redis::save' => ['bool'],
 'Redis::scan' => ['array<int,string>|false', '&rw_iterator'=>'?int', 'pattern='=>'?string', 'count='=>'?int'],
 'Redis::sCard' => ['int', 'key'=>'string'],
 'Redis::sContains' => ['', 'key'=>'string', 'value'=>'string'],
 'Redis::script' => ['mixed', 'command'=>'string', 'script'=>'string'],
 'Redis::sDiff' => ['array', 'key1'=>'string', 'key2'=>'string', 'keyN='=>'string'],
-'Redis::sDiffStore' => ['int', 'dstKey'=>'string', 'key1'=>'string', 'key2'=>'string', 'keyN='=>'string'],
+'Redis::sDiffStore' => ['int|false', 'dstKey'=>'string', 'key1'=>'string', 'key2'=>'string', 'keyN='=>'string'],
 'Redis::select' => ['bool', 'dbindex'=>'int'],
 'Redis::sendEcho' => ['string', 'msg'=>'string'],
 'Redis::set' => ['bool', 'key'=>'string', 'value'=>'string', 'options='=>'array'],
 'Redis::set\'1' => ['bool', 'key'=>'string', 'value'=>'string', 'timeout='=>'int'],
 'Redis::setBit' => ['int', 'key'=>'string', 'offset'=>'int', 'value'=>'int'],
 'Redis::setEx' => ['bool', 'key'=>'string', 'ttl'=>'int', 'value'=>'string'],
+'Redis::setex' => ['bool', 'key'=>'string', 'ttl'=>'int', 'value'=>'string'],
```

Comment:
Remove this, this is the same case-insensitive method name as setEx. Same for setnx